tests/Resources/ti.ui.tableview.addontest.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.UI.TableView', function () {
+
+	it('get TableViewRow rect with height set to Ti.UI.SIZE', function (finish) {
+		var win = Ti.UI.createWindow(),
+			table = Ti.UI.createTableView({ width: Ti.UI.FILL, height: Ti.UI.FILL }),
+			row = Ti.UI.createTableViewRow({ width: Ti.UI.FILL, height: Ti.UI.SIZE }),
+			view = Ti.UI.createView({ height: 150 });
+		row.add(view);
+		table.setData([row]);
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/Resources/t..., tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.tableview.addontest.js;
  Line: 20;
-->

- :warning: tests/Resources/ti.ui.tableview.addontest.js line 20 – A space is required after '['. (array-bracket-spacing)
- :warning: tests/Resources/ti.ui.tableview.addontest.js line 20 – A space is required before ']'. (array-bracket-spacing)